tools/build/build.js

```diff
@@ -327,7 +327,7 @@ export const TguiTarget = new Juke.Target({
     'tgui/.yarn/install-target',
     'tgui/webpack.config.js',
     'tgui/**/package.json',
-    'tgui/packages/**/*.+(js|cjs|ts|tsx|scss)',
+    'tgui/packages/**/*.+(js|cjs|ts|tsx|jsx|scss)',
```

Comment:
We can remove 'js' because they were all replaced with 'jsx' in  #79877 and #79894 also not sure why we have 'ts' when 'tsx' is the correct extension for typescript files.

@jlsnow301 thoughts?

Reply:
They might exist in one way or another (e.g. untyped polyfills), so I wouldn't remove it.